File: php/fizzbuzz/FizzBuzzSequence.php
<?php

require_once 'FizzBuzz.php';

class FizzBuzzSequence {

    private $_fizzBuzz;

    function __construct() {
        $this->_fizzBuzz = new FizzBuzz();
    }

    public function getFizzBuzzSequence($start, $stop) {
        $arr = array();
        $calculatedString = "";

        for ($i = $start; $i <= $stop; $i++) {
            $arr[] = $this->_fizzBuzz->getFizzBuzzValueOf($i);
        }

        $calculatedString = implode(",", $arr);

        return $calculatedString;
    }

}

File: java/fizzbuzz/src/fizzbuzz/FizzBuzzSequence.java
package fizzbuzz;

public class FizzBuzzSequence {

    private FizzBuzz _fizzBuzz;

    public FizzBuzzSequence() {
        _fizzBuzz = new FizzBuzz();
    }

    public String getFizzBuzzSequence(int start, int stop) {
        String calculatedString = "";
        String arr[] = new String[stop - start + 1];

        for (int i = start; i <= stop; i++) {
            arr[i-1] = _fizzBuzz.getFizzBuzzValueOf(i);
        }
        calculatedString = FizzBuzzSequence. implode(arr, ",");

        return calculatedString;
    }

    /**
     * Helperfunktion - Wandelt die Inhalte eines Arrays in eine String um
     * @param ary Array, welcher umgewandelt werden soll
     * @param delim Trennzeichen zwischen den einzelnen Elementen
     * @return Umgewandelter String
     */
    public static String implode(String[] ary, String delim) {
        String out = "";
        for (int i = 0; i < ary.length; i++) {
            if (i != 0) {
                out += delim;
            }
            out += ary[i];
        }
        return out;
    }
}

